### ISSUE TITLE
When will dataset be published?

### ISSUE THREAD

**Issue by MoritzLaurer** [facebookresearch/dadc-limit#3]
Hey, super interesting paper and data! When will it be uploaded?

**Comment by douwekiela** [comment]
Thanks! We hope to do so soon :) Probably slightly after the ARR ACL deadline.

**Comment by Eric-Wallace** [comment]
Yep, ill post the data and code this week! Thanks for pinging on this.
